packages/style-loader/index.js

```diff
@@ -1,10 +1,66 @@
-module.exports = ({ config }, options = {}) => config.module
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const merge = require('deepmerge');
+const { pick, mapValues } = require('lodash');
+
+module.exports = (neutrino, opts = {}) => {
+  const isBuild = neutrino.options.command === 'start';
+  const options = merge({
+    styleUseId: 'style',
+    cssHotLoaderUseId: 'cssHotLoader',
```

Comment:
How would you feel about this just being `hot`, similar to the `hot` plugin?

Reply:
Sure. Do you mean the the option name (`cssHotLoaderUseId`) or the value `'cssHotLoader'`?

Not sure if we even need it configurable as an option, was just following suit with `options.styleUseId`, `options.cssUseId`